src/pages/LendingTeam/TeamLeaderboards.vue

```diff
@@ -0,0 +1,229 @@
+					>
+						<img v-if="team.image && team.image.url"
+							class="tw-w-5
+							tw-h-5
+							tw-flex-none
+							tw-mb-1"
+							:src="team.image.url"
+						>
+
+						<img v-else class="tw-w-5
+							tw-h-5
+							tw-flex-none
+							tw-mb-1"
+							:src="teamNoImage"
+						>
+
+						<div class="tw-flex
+						tw-flex-row
+						tw-flex-wrap
+						tw-pl-1
+						tw-flex-grow
+						tw-overflow-hidden
+						tw-whitespace-nowrap"
+						>
+							<div class="tw-flex
+							tw-flex-nowrap
+							tw-flex-none
+							tw-w-full"
+							>
+								<div class="tw-flex-1 tw-overflow-hidden tw-text-ellipsis tw-text-base">
+									<router-link :to="`/team/${team.teamPublicId}`"
+										state=""
+										class=""
+									>
+										<div class="tw-text-base
+										tw-text-primary
+										tw-overflow-hidden
+										tw-text-ellipsis"
+										>
+											{{ team.name }}
+										</div>
+									</router-link>
+								</div>
+								<div class="tw-flex-none tw-ml-1">
+									<template v-if="!isNewMembers">
+										{{ numeral(team.lentAmount).format('$0,0a') }}
+									</template>
+									<template v-else>
+										{{ numeral(team.lenderCount).format('0,0a') }}
+									</template>
+								</div>
+							</div>
+							<div class="tw-flex-none
+							tw-w-full"
+							>
+								<template v-if="!isNewMembers">
+									<kv-progress-bar
+										aria-label="Total Amount the Team has lent"
+										:min="0"
+										max="75000000"
+										:value="team.lentAmount"
+									/>
+								</template>
+								<template v-else>
+									<kv-progress-bar
+										aria-label="Total Amount of Members the Team has"
+										:min="0"
+										max="200000"
+										:value="team.lenderCount"
+									/>
+								</template>
+							</div>
+						</div>
+					</div>
+				</kv-tab-panel>
+			</template>
+		</kv-tabs>
+	</div>
+</template>
+
+<script>
+import KvProgressBar from '@/components/Kv/KvProgressBar';
+import numeral from 'numeral';
+import teamNoImage from '@/assets/images/team_s135.png';
+import { fetchLeaderboard } from '../../util/teamsUtil';
+import KvTab from '~/@kiva/kv-components/vue/KvTab';
+import KvTabPanel from '~/@kiva/kv-components/vue/KvTabPanel';
+import KvTabs from '~/@kiva/kv-components/vue/KvTabs';
+
+export default {
+	name: 'TeamLeaderboards',
+	components: {
+		KvProgressBar,
+		KvTabs,
+		KvTab,
+		KvTabPanel,
+	},
+	inject: ['apollo'],
+	data() {
+		return {
+			teams: ['10'],
+			numeral,
+			teamNoImage
+		};
+	},
+	props: {
+		boardType: {
+			type: String,
+			required: true,
+		}
+	},
+	computed: {
+		isNewMembers() {
+			return this.boardType === 'memberCount';
+		}
+	},
+	options: {
+		limit: 10
+	},
+	methods: {
+		handleTabChanged(index) {
+			console.log(index);
+		},
+	},
+	mounted() {
+		fetchLeaderboard(this.apollo, this.boardType).then(teams => {
+			this.teams = teams?.values ?? [];
+		});
+	}
+};
+</script>
+
+<style lang="scss" scoped>
```

Comment:
We are moving away from scss. Are these copied from elsewhere? Or can we convert to tailwind?